lnclient/lnd/lnd.go

```diff
@@ -1219,12 +1219,12 @@ func (svc *LNDService) GetOnchainBalance(ctx context.Context) (*lnclient.Onchain
 	}, nil
 }
 
-func (svc *LNDService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, sendAll bool) (txId string, err error) {
+func (svc *LNDService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, feeRate uint64, sendAll bool) (txId string, err error) {
 	resp, err := svc.client.SendCoins(ctx, &lnrpc.SendCoinsRequest{
-		Addr:       toAddress,
-		SendAll:    sendAll,
-		Amount:     int64(amount),
-		TargetConf: 1,
+		Addr:        toAddress,
+		SendAll:     sendAll,
+		Amount:      int64(amount),
+		SatPerVbyte: feeRate,
```

Comment:
I think `TargetConf: 1` should be set instead of SatPerVbyte unless the explicitly sets the fee rate.